charts/opentelemetry-operator/UPGRADING.md

```diff
@@ -1,5 +1,25 @@
 # Upgrade guidelines
 
+## 0.56.0 to 0.57.0
+
+This Chart now installs CRDs as templates. OpenTelemetry Operator is rolling out a new version of the OpenTelemetryCollector CRD. For information
+about this change, see the [following document](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/crd-changelog.md#opentelemetrycollectoropentelemetryiov1beta1).
+
+This change includes a conversion webhook, which needs to reference a namespaced webhook Service, and therefore needs to include the release namespace. See [the following issue](https://github.com/open-telemetry/opentelemetry-helm-charts/issues/1167) for more information on the CRD change.
+
+As a result, manual steps are necessary to convince Helm to manage existing CRDs. This involves adding some annotations and labels. Set `RELEASE_NAME` and `RELEASE_NAMESPACE` to the values you're using for your Helm release, respectively:
+
+```bash
```

Comment:
we should specify that this should be done _prior_ to upgrade

Reply:
You can't actually upgrade unless you do this, Helm will complain and fail.

Reply:
yeah all the more reason to say that very loudly in the doc ha